cypress/integration/loans/renewal-failure.spec.js

```diff
@@ -0,0 +1,201 @@
+            permanentLocationId: Cypress.env(CY_ENV.LOCATION)[0].id,
+            sourceId: Cypress.env(CY_ENV.HOLDING_SOURCES)[0].id,
+          }],
+          items: [[{
+            barcode: itemData.barcode,
+            status: { name: 'Available' },
+            permanentLoanType: { id: Cypress.env(CY_ENV.LOAN_TYPES)[0].id },
+            materialType: { id: materialTypeId },
+          }]],
+        });
+      })
+      // create loan policy
+      .then(() => {
+        cy.createLoanPolicy({
+          name: loanPolicyData.name,
+          id: loanPolicyData.id,
+          loanable: true,
+          loansPolicy: {
+            closedLibraryDueDateManagementId: LIBRARY_DUE_DATE_MANAGMENT.CURRENT_DUE_DATE,
+            period: {
+              duration: 3,
+              intervalId: 'Weeks',
+            },
+            profileId: LOAN_PROFILE.ROLLING,
+          },
+          renewable: false,
+          renewalsPolicy: {
+            unlimited: true,
+          },
+        });
+      })
+      // create circulation rules
+      .then(() => {
+        const requestPolicyId = Cypress.env(CY_ENV.REQUEST_POLICY)[0].id;
+        const noticePolicyId = Cypress.env(CY_ENV.NOTICE_POLICY)[0].id;
+        const overdueFinePolicyId = Cypress.env(CY_ENV.OVERDUE_FINE_POLICY)[0].id;
+        const lostItemFeesPolicyId = Cypress.env(CY_ENV.LOST_ITEM_FEES_POLICY)[0].id;
+        const policy = `l ${loanPolicyData.id} r ${requestPolicyId} n ${noticePolicyId} o ${overdueFinePolicyId} i ${lostItemFeesPolicyId}`;
+        const newRule = `priority: number-of-criteria, criterium (t, s, c, b, a, m, g), last-line\nfallback-policy: ${policy}\nm ${materialTypeId}: ${policy}`;
+
+        cy.updateCirculationRules({
+          rulesAsText: newRule,
+        });
+      })
+      // checkout item
+      .then(() => {
+        cy.createItemCheckout({
+          servicePointId,
+          itemBarcode: itemData.barcode,
+          userBarcode: renewUserData.barcode,
+        })
+          .then(checkedItem => {
+            loanId = checkedItem.body.id;
+          });
+      });
+  });
+
+  after(() => {
+    cy.createItemCheckinApi({
+      itemBarcode: itemData.barcode,
+      servicePointId,
+      checkInDate: moment.utc().format(),
+    })
+      .then(() => {
+        cy.deleteUser(renewUserData.id);
+        cy.deleteUser(renewOverrideUserId);
+        cy.getInstance({ limit: 1, expandAll: true, query: `"items.barcode"=="${itemData.barcode}"` })
+          .then((instance) => {
+            cy.deleteItem(instance.items[0].id);
+            cy.deleteHoldingRecord(instance.holdings[0].id);
+            cy.deleteInstanceApi(instance.id);
+          });
+        cy.updateCirculationRules({
+          rulesAsText: initialCircRules,
+        });
+        cy.deleteLoanPolicy(LOAN_POLICY_ID);
+      });
+  });
+
+  it('C568 Renewal: failure because loan is not renewable', { tags: [TestType.smoke] }, () => {
```

Comment:
I haven't found checking of "** No Override button displays" from step1 in testrail(

Reply:
Yes, It doesn't exist. I have some difficulties with it. 
User doesn't have permission to see 'Override' button but button exists. If I create the user with the same permissions manually button has correct behavior and doesn't exist. I do not know why it happens and how to handle it. 
And I will appreciate it if someone can help with it or suggest something.

Also, I asked about it in QA channel:
![image](https://user-images.githubusercontent.com/42437614/167165411-fd440cf3-3e6a-4dd5-aa72-dceed8571ca6.png)

Reply:
can't see related todo